src/Symfony/Component/EventDispatcher/composer.json

```diff
@@ -25,6 +25,7 @@
         "symfony/config": "^5.4|^6.0|^7.0",
         "symfony/error-handler": "^5.4|^6.0|^7.0",
         "symfony/http-foundation": "^5.4|^6.0|^7.0",
+        "symfony/http-kernel": "^6.4",
```

Comment:
```suggestion
        "symfony/http-kernel": "^6.4|^7.0",
```

Reply:
I think I can just drop this now that the `VirtualRequestStack` is not type-hinted anymore.